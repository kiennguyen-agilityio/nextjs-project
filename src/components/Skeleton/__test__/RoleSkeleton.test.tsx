import { render } from '@testing-library/react';

// component
import RoleSkeleton from '@/components/Skeleton/RoleSkeleTon';
describe('RoleSkeleton Component', () => {
  it('renders without crashing and matches snapshot', () => {
    const { asFragment } = render(<RoleSkeleton />);
    expect(asFragment()).toMatchSnapshot();
  });
});
