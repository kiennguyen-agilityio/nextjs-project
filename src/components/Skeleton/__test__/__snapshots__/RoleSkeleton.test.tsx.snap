// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RoleSkeleton Component renders without crashing and matches snapshot 1`] = `
<DocumentFragment>
  <table
    class="min-w-full bg-white border border-gray-200"
  >
    <thead>
      <tr>
        <th
          class="py-2 px-4 border-b bg-gray-100 w-[5%]"
        >
          <div
            class="h-4 bg-gray-300 rounded-md animate-pulse"
          />
        </th>
        <th
          class="py-2 px-4 border-b bg-gray-100"
        >
          <div
            class="h-4 bg-gray-300 rounded-md animate-pulse"
          />
        </th>
        <th
          class="py-2 px-4 border-b bg-gray-100"
        >
          <div
            class="h-4 bg-gray-300 rounded-md animate-pulse"
          />
        </th>
        <th
          class="py-2 px-4 border-b bg-gray-100"
        />
      </tr>
    </thead>
    <tbody>
      <tr
        class="text-center hover:bg-gray-100"
      >
        <td
          class="py-2 px-4 border-b"
        >
          <div
            class="h-4 bg-gray-300 rounded-md animate-pulse"
          />
        </td>
        <td
          class="py-2 px-4 border-b"
        >
          <div
            class="h-4 bg-gray-300 rounded-md animate-pulse"
          />
        </td>
        <td
          class="py-2 px-4 border-b"
        >
          <div
            class="h-4 bg-gray-300 rounded-md animate-pulse"
          />
        </td>
        <td
          class="py-2 px-4 border-b"
        >
          <div
            class="w-6 h-6 bg-gray-300 rounded-full animate-pulse"
          />
        </td>
      </tr>
      <tr
        class="text-center hover:bg-gray-100"
      >
        <td
          class="py-2 px-4 border-b"
        >
          <div
            class="h-4 bg-gray-300 rounded-md animate-pulse"
          />
        </td>
        <td
          class="py-2 px-4 border-b"
        >
          <div
            class="h-4 bg-gray-300 rounded-md animate-pulse"
          />
        </td>
        <td
          class="py-2 px-4 border-b"
        >
          <div
            class="h-4 bg-gray-300 rounded-md animate-pulse"
          />
        </td>
        <td
          class="py-2 px-4 border-b"
        >
          <div
            class="w-6 h-6 bg-gray-300 rounded-full animate-pulse"
          />
        </td>
      </tr>
      <tr
        class="text-center hover:bg-gray-100"
      >
        <td
          class="py-2 px-4 border-b"
        >
          <div
            class="h-4 bg-gray-300 rounded-md animate-pulse"
          />
        </td>
        <td
          class="py-2 px-4 border-b"
        >
          <div
            class="h-4 bg-gray-300 rounded-md animate-pulse"
          />
        </td>
        <td
          class="py-2 px-4 border-b"
        >
          <div
            class="h-4 bg-gray-300 rounded-md animate-pulse"
          />
        </td>
        <td
          class="py-2 px-4 border-b"
        >
          <div
            class="w-6 h-6 bg-gray-300 rounded-full animate-pulse"
          />
        </td>
      </tr>
      <tr
        class="text-center hover:bg-gray-100"
      >
        <td
          class="py-2 px-4 border-b"
        >
          <div
            class="h-4 bg-gray-300 rounded-md animate-pulse"
          />
        </td>
        <td
          class="py-2 px-4 border-b"
        >
          <div
            class="h-4 bg-gray-300 rounded-md animate-pulse"
          />
        </td>
        <td
          class="py-2 px-4 border-b"
        >
          <div
            class="h-4 bg-gray-300 rounded-md animate-pulse"
          />
        </td>
        <td
          class="py-2 px-4 border-b"
        >
          <div
            class="w-6 h-6 bg-gray-300 rounded-full animate-pulse"
          />
        </td>
      </tr>
      <tr
        class="text-center hover:bg-gray-100"
      >
        <td
          class="py-2 px-4 border-b"
        >
          <div
            class="h-4 bg-gray-300 rounded-md animate-pulse"
          />
        </td>
        <td
          class="py-2 px-4 border-b"
        >
          <div
            class="h-4 bg-gray-300 rounded-md animate-pulse"
          />
        </td>
        <td
          class="py-2 px-4 border-b"
        >
          <div
            class="h-4 bg-gray-300 rounded-md animate-pulse"
          />
        </td>
        <td
          class="py-2 px-4 border-b"
        >
          <div
            class="w-6 h-6 bg-gray-300 rounded-full animate-pulse"
          />
        </td>
      </tr>
    </tbody>
  </table>
</DocumentFragment>
`;
